dbms/src/DataStreams/FilterTransformAction.h

```diff
@@ -15,6 +15,7 @@
 #pragma once
 
 #include <Columns/FilterDescription.h>
+#include <DataStreams/IBlockInputStream.h>
```

Comment:
I tend to use `using FilterPtr = IColumn::Filter *;` instead of `#include <DataStreams/IBlockInputStream.h>`.